client/wfm/src/process-instance/ProcessInstanceRepository.ts

```diff
@@ -0,0 +1,8 @@
+import {ProcessInstance} from './ProcessInstance';
+export interface ProcessInstanceRepository {
+  getAll(): Promise<ProcessInstance[]>;
+  getById(id: string): Promise<ProcessInstance>;
+  getByProcessId(id: string): Promise<ProcessInstance[]>;
```

Comment:
Used [here](https://github.com/feedhenry-raincatcher/raincatcher-core/blob/vehicle-inspection-example/integration/vehicle-inspection/src/server/index.ts#L62-L66), but mostly for convenience/completeness. I think the POST method is more important there.

We can also move it to `POST /[process-]instances`, but we'd probably have to include the process id in the payload.